libs/vectorise/include/vectorise/memory/endian.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
imho, this should go to platform.hpp, as it already has helper functions for big/little endian